src/components/PriceCard.vue

```diff
@@ -37,7 +37,13 @@
             <p>
               <span>{{ getPriceValueDisplay() }}</span>
               <span v-if="hasProductQuantity"> ({{  getPricePerKilo() }})</span>
-              <span> on <i>{{ getDateFormatted(price.date) }}</i></span>
+              <span>
+                <i18n-t keypath="PriceCard.PriceDate" tag="p">
```

Comment:
why the `tag="p"` ? I see it elsewhere, it's making new lines

Reply:
fixed here : https://github.com/openfoodfacts/open-prices-frontend/commit/b5d4282cebe574f6d973d076489c51196d92a25d